justfile

```
@@ -19,6 +19,9 @@ cargo := "cargo" + if toolchain != "" { " +" + toolchain } else { "" }
 # The version name to use for packages.
 package_version := `git rev-parse --short HEAD`
 
+# Default docker tag name
+default_docker_tag := "gchr.io/linkerd-io/proxy:" + env_var_or_default("USER", "dev") + "-" + package_version
```

What is `linkerd-io`?

I chose not to do something like this originally because we definitely do not want to push these images to the container registry.

Perhaps better to do something like
```suggestion
default_docker_tag := env_var_or_default("USER", "dev") + "/l2-proxy:" + package_version
```

Also, AFAICT this variable is never used. Maybe we should stop taking the tag as a recipe argument and instead always use the variable?

---

Sure, I'm fine with using a local image tag that differs from the published one, I wasn't sure what the preference would be.

---

> Also, AFAICT this variable is never used. Maybe we should stop taking the tag as a recipe argument and instead always use the variable?

agh, that's my fault, i must have messed up as a rebase. it was passed as the default value for the `tag` argument in the `docker` recipe. i would also be fine with just always using the variable, since currently, if you want to pass other arguments to `just docker` (such as a mode), you would have to override the tag name as well...